tests/spcs/test_image_repository.py

```diff
@@ -131,3 +131,43 @@ def test_list_tags(
     assert json.loads(result.output) == [
         {"tag": "DB/SCHEMA/IMAGES/super-cool-repo:1.2.0"}
     ]
+
+
+@mock.patch(
+    "snowflake.cli.plugins.spcs.image_repository.commands.ImageRepositoryManager._execute_query"
+)
+@mock.patch(
+    "snowflake.cli.plugins.spcs.image_repository.commands.ImageRepositoryManager._conn"
+)
+def test_get_repository_url_cli(mock_conn, mock_execute_query, runner, mock_cursor):
+    mock_conn.database = "DB"
+    mock_conn.schema = "SCHEMA"
+    mock_conn.role = "MY_ROLE"
```

Comment:
Are database, schema and role needed?

Reply:
good point, they were previously needed for get_repository_url_list but are no longer necessary